Add Prometheus Links Documentation + Fix Broken Links

jasonBirchall opened the pull request:
Connects to https://github.com/ministryofjustice/cloud-platform/issues/409
& 
Connects to https://github.com/ministryofjustice/cloud-platform/issues/405

**WHAT**
1. Fix the broken links in the Kibana documentation:
https://ministryofjustice.github.io/cloud-platform-user-docs/03-other-topics/004-kibana/#how-to-access-my-log-data-using-kibana
2. Adds a document to the user docs called `Using the Cloud Platform Prometheus, AlertManager and Grafana`.
3. Fixes the broken links in the `Kubernetes basics` docs:
https://ministryofjustice.github.io/cloud-platform-user-docs/03-other-topics/005-k8s-basics/#kubernetes-basics---links-to-helpful-beginner-resources

**WHY**
1. This will allow users to click the hyperlinks and redirect to the relevant page. I found this issue whilst attempting to fix another ticket. 
2. As users want to know how to access the Cloud Platform Prometheus, AlertManager or Grafana. They can find guidance on this in the user guide. This links to Connects to https://github.com/ministryofjustice/cloud-platform/issues/409.
3. It was pointed out by users that the `Kubernetes basics` docs has a broken link. This link has now been fixed under Connects to https://github.com/ministryofjustice/cloud-platform/issues/405.

Commit:
Fixed broken link in kibana documentation

This was noticed as part of ticket #409

Commit:
Add first draft of Prometheus documentation

https://github.com/ministryofjustice/cloud-platform/issues/409

Commit:
Fix broken links in basic kubectl docs

razvan-moj-zz commented on 03-other-topics/004-kibana.md:
```diff
@@ -4,16 +4,16 @@ This document is intended to assist engineers in accessing application and syste
 ## How do I use Kibana?
 
-https://www.elastic.co/guide/en/kibana/6.3/search.html
+[https://www.elastic.co/guide/en/kibana/6.3/search.html](https://www.elastic.co/guide/en/kibana/6.3/search.html)
```
cluster is 6.2 right now, and there are breaking changes in 6.3 so maybe pin to the exact release: https://www.elastic.co/guide/en/elasticsearch/reference/current/release-notes-6.2.3.html

Commit:
Remove test-1 reference in kibana docs

After a discussion with a colleague, it was decided this was no longer required.
